### PR TITLE
SCMOD-13610: Release 3.5

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-13610

### CODE REVIEW

**Review by DavidGerald-Milligan — APPROVED** [review]

File: release-notes-1.9.0.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
 
 #### Known Issues
+
+- None
```
Comment (DavidGerald-Milligan) [inline release-notes-1.9.0.md]: I think this needs renamed to release-notes-1.8.2.md

File: release-notes-1.8.1.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#### Version Number
```
Comment (DavidGerald-Milligan) [inline release-notes-1.8.1.md]: release-notes-1.8.2.md?

File: release-notes-1.9.0.md (hunk 1 above)
Comment (xbreizh) [reply release-notes-1.9.0.md]: done

File: release-notes-1.8.1.md (hunk 2 above)
Comment (xbreizh) [reply release-notes-1.8.1.md]: true, done

File: release-notes-1.8.2.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#### Version Number
+${version-number}
+
+#### New Features
+- SCMOD-12326: Updated to use Artifactory for private Docker images
```
Comment (michael-bryson) [inline release-notes-1.8.2.md:5]: This is a patch release, it cannot have any new features. I also would not call this out in the release notes, if you have nothing else to put in here simply add "This release includes OS package updates only.", for example [https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2](https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2)

**Pull request merged**